specifications/client/verifying_client.md

```diff
@@ -0,0 +1,125 @@
+# Diem Verifying Client Spec
+
+THIS IS A WORK IN PROGRESS
+
+Links:
+
+* https://github.com/novifinancial/libra-specs/blob/master/client/client.md#updatetolatestledger-requests
+* https://github.com/novifinancial/libragogo/blob/master/client_specification.md#submittransaction-request
+* https://github.com/diem/diem/blob/master/json-rpc/json-rpc-spec.md
+
+## Data serialisation
```

Comment:
might want to stay consistent with the other specs and use the US spelling here (serialization)